app/lib/actions/katello/content_view_version/incremental_update.rb

```diff
@@ -138,19 +134,11 @@ def repos_to_copy(old_version, new_components)
           end
         end
 
-        def copy_repos(new_repo, new_version, content, dep_solve)
-          copy_output = []
+        def copy_repos(new_repo)
           sequence do
-            solve_dependencies = new_version.content_view.solve_dependencies || dep_solve
-            copy_output += copy_deb_content(new_repo, solve_dependencies, content[:deb_ids])
-            copy_output += copy_yum_content(new_repo, solve_dependencies,
-                                            content[:package_ids],
-                                            content[:errata_ids])
-
             plan_action(Katello::Repository::MetadataGenerate, new_repo)
```

Comment:
@jlsherrill Since we are removing the unit copy here for pulp2 - are the following metadata generation and indexing required?

Reply:
It looks like today there are two branches in this code: 
* pulp3 with yum
* everything else
 
and really the only thing supported with everything else is
  * pulp2 w/yum
  * pulp2 w/ deb
Which means that pulp3 w/deb is not supported via inc update, and no other content types are (which is expected). 

However, reading the pulp3 with yum branch, i'm not sure where the publications are getting created, or the content is getting indexed?  Maybe i'm missing something...

Reply:
I think what you've got is fine, let me file a card to investigate this